### DISCUSSION TITLE
Function to generate the funfact at round end from a CS:GO demofiles cs_win_panel_round event

### DISCUSSION THREAD

**Discussion by zuloo** [saul/demofile#338] [Show and tell]
https://gist.github.com/zuloo/84bbace7ea850ec84de60b4d9ea975bb

**Comment by saul** [comment]
I like this - this is really cool 😄 I've just released v2.3.0 which adds a `player` property to the `cs_win_panel_round` event (amongst others, see the release notes) to help make your code a bit simpler.
